build/repo.targets

```diff
@@ -1,43 +0,0 @@
-<Project InitialTargets="TagCiBuilds">
-  <Target Name="TagCiBuilds" Condition=" '$(CI)' == 'true' AND '$(BUILD_REASON)' != 'PullRequest' ">
-    <Message Importance="high" Text="##vso[build.addbuildtag]daily-build" Condition=" '$(IsFinalBuild)' != 'true' " />
-    <Message Importance="high" Text="##vso[build.addbuildtag]release-candidate" Condition=" '$(IsFinalBuild)' == 'true' " />
-  </Target>
-
-  <Target Name="PublishToBAR">
-    <MSBuild Projects="$(MSBuildThisFileDirectory)..\eng\tools.proj"
-             Targets="Restore"
-             Properties="$(BuildProperties);RestorePackagesPath=$(RepositoryRoot).packages\" />
-
-    <MSBuild Projects="$(MSBuildThisFileDirectory)..\eng\publish.proj"
-             Targets="Publish"
-             Properties="$(BuildProperties);RestorePackagesPath=$(RepositoryRoot).packages\" />
-  </Target>
-
-  <Target Name="GenerateProjectList" DependsOnTargets="ResolveProjects">
```

Comment:
We need this target still. It's used to generate the eng/ProjectReferences.props file. Is there an Arcade equivalent of repo targets?

Reply:
There's no repo.targets equivalent, but we can just do "dotnet msbuild repo.targets /t:GenerateProjectList". I made some tweeks and tested it out locally and it works (although it's not as "clean" as having the target already be there.)